XPlat/Shopify/iOS/source/Shopify.iOS/ApiDefinition.cs

```diff
@@ -0,0 +1,1964 @@
+	}
+
+	// @interface BUYLineItem : BUYObject <BUYSerializable>
+	[BaseType (typeof(Object), Name = "BUYLineItem")]
+	interface LineItem : Serializable
+	{
+		// @property (readonly, nonatomic, strong) NSString * lineItemIdentifier;
+		[Export ("lineItemIdentifier", ArgumentSemantic.Strong)]
+		string LineItemIdentifier { get; }
+
+		// @property (readonly, nonatomic, strong) NSNumber * variantId;
+		[Export ("variantId", ArgumentSemantic.Strong)]
+		NSNumber VariantId { get; }
+
+		// @property (readonly, nonatomic, strong) NSNumber * productId;
+		[Export ("productId", ArgumentSemantic.Strong)]
+		NSNumber ProductId { get; }
+
+		// @property (nonatomic, strong) NSDecimalNumber * quantity;
+		[Export ("quantity", ArgumentSemantic.Strong)]
+		NSDecimalNumber Quantity { get; set; }
+
+		// @property (readonly, nonatomic, strong) NSDecimalNumber * grams;
+		[Export ("grams", ArgumentSemantic.Strong)]
+		NSDecimalNumber Grams { get; }
+
+		// @property (nonatomic, strong) NSDecimalNumber * price;
+		[Export ("price", ArgumentSemantic.Strong)]
+		NSDecimalNumber Price { get; set; }
+
+		// @property (nonatomic, strong) NSDecimalNumber * linePrice;
+		[Export ("linePrice", ArgumentSemantic.Strong)]
+		NSDecimalNumber LinePrice { get; set; }
+
+		// @property (readonly, nonatomic, strong) NSDecimalNumber * compareAtPrice;
+		[Export ("compareAtPrice", ArgumentSemantic.Strong)]
+		NSDecimalNumber CompareAtPrice { get; }
+
+		// @property (copy, nonatomic) NSString * title;
+		[Export ("title")]
+		string Title { get; set; }
+
+		// @property (copy, nonatomic) NSString * variantTitle;
+		[Export ("variantTitle")]
+		string VariantTitle { get; set; }
+
+		// @property (nonatomic, strong) NSNumber * requiresShipping;
+		[Export ("requiresShipping", ArgumentSemantic.Strong)]
+		NSNumber RequiresShipping { get; set; }
+
+		// @property (readonly, copy, nonatomic) NSString * sku;
+		[Export ("sku")]
+		string Sku { get; }
+
+		// @property (readonly, assign, nonatomic) BOOL taxable;
+		[Export ("taxable")]
+		bool Taxable { get; }
+
+		// @property (copy, nonatomic) NSDictionary * properties;
+		[Export ("properties", ArgumentSemantic.Copy)]
+		NSDictionary Properties { get; set; }
+
+		// @property (readonly, copy, nonatomic) NSString * fulfillmentService;
+		[Export ("fulfillmentService")]
+		string FulfillmentService { get; }
+
+		// -(instancetype)initWithVariant:(BUYProductVariant *)variant;
+		[Export ("initWithVariant:")]
+		IntPtr Constructor (ProductVariant variant);
+	}
+
+	// @interface BUYCartLineItem : BUYLineItem
+	[BaseType (typeof(LineItem), Name = "BUYCartLineItem")]
+	interface CartLineItem
+	{
+		// @property (readonly, nonatomic, strong) BUYProductVariant * variant;
+		[Export ("variant", ArgumentSemantic.Strong)]
+		ProductVariant Variant { get; }
+	}
+
+	// @interface BUYTheme : NSObject
+	[BaseType (typeof(NSObject), Name = "BUYTheme")]
+	interface Theme
+	{
+		// extern const CGFloat kBuyPaddingSmall;
+		[Static]
```

Comment:
[Static] attribute is not necessary because, IIRC, Fields are bound as static parameters.